### DISCUSSION TITLE
Command 'chia' not found

### DISCUSSION THREAD

**Discussion by xiaogan18** [Chia-Network/chia-blockchain#4761] [Support]
When i exec `chia init`, i got this:
```
(venv) ubuntu@VM-0-11-ubuntu:~/chia-blockchain$ chia init

Command 'chia' not found, did you mean:

  command 'chi2' from deb paml

Try: sudo apt install <deb name>
```
I have already exec `source venv/bin/activate` first.

**Comment by langubtc** [comment]
cd chia-blockchain && . ./activate

**Comment by ghost** [comment]
(venv) indicates that you're already in the virtual Python environment, so the *chia* command should work.

Did you install it according to the [INSTALL document](https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian)? Especially the *sh install.sh* part?

```
git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules
cd chia-blockchain

sh install.sh

. ./activate
```

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
